reviewrot/basereview.py

```diff
@@ -195,15 +210,15 @@ def format_duration(created_at):
     def since(self):
         return self.format_duration(created_at=self.time)
 
-    def __str__(self):
-        return self.format('oneline')
-
     def format(self, style, i, N):
-        """ Format the result in a given style.
-
-        style - the name of the style.
-        i - an integer, indicating the position in a list.
-        N - the total size of the list.
+        """
+        Format the result in a given style.
+        Args:
+            style(str): the name of the style.
+            i(int): an integer, indicating the position in a list.
```

Comment:
You could just write, 'position in a list' or 'index' or something simpler and not mention data type again in desc